support-e2e/tests/utils/testUserDetails.ts

```diff
@@ -18,31 +19,30 @@ export const setTestUserRequiredDetails = async (
 
 export const setTestUserDetails = async (
 	page: Page,
-	testDetails: TestDetails,
```

Comment:
There are fields in `TestDetails` which this function doesn't care about, so I've changed to interface. This allows us to use this method elsewhere, only providing the data needed.

Reply:
Don't see any optional parameters but probably saves having to build that whole confusingly named `TestDetails` object? Plus more concise use of (shorter)  field names within this function is a bonus.